index.js

```diff
@@ -73,8 +74,7 @@ server.register(require('inert'), function (err) {
       };
       helpers.generate(options, function (result) {
         reply(
-          JSON // jshint ignore:line
-            .stringify(result)
+          JSON.stringify(result)    // jshint ignore:line
```

Comment:
Unfortunately, yes. Your JSHint options are weird :/

`enforceall` is deprecated, and in theory JSON should be defined...